.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: instrumentation_tests
+
+on: [push, pull_request]
+
+env:
+  GRADLE_OPTS: "-Dorg.gradle.jvmargs=-Xmx4g -Dorg.gradle.daemon=false -Dkotlin.incremental=false"
+
+jobs:
+  android:
+    runs-on: macos-latest
+
+    strategy:
+      fail-fast: true
+      matrix:
+        api-level:
+          - 21
+          - 24
+          - 26
+          - 29
```

Comment:
Let's just go with the latest and greatest API level - it'll speed up the feedback loop and the E2E tests already have sufficient coverage for us to know if something went wrong on a specific version.

Reply:
Done - 29 appears to be the latest available with that action.